OWDTestToolkit/utils/decorators.py

```diff
@@ -31,7 +31,7 @@ def func_with_retries(*args, **kwargs):
                     res = func_to_retry(*args, **kwargs)
                     return res
                 except Exception, e:
-                    utils.reporting.debug(">>>>>> The function to retry failed: {}".format(e))
+                    utils.reporting.debug(">>>>>> The function to retry failed")
```

Comment:
I can't remember why I did this (this branch has been opened too long :( ), but I believe I changed this bcs there was something really weird. But, since I don't remember, I'll leave as it was before. It's preferrable to see the "petardazo" rather than not seeing anything at all :+1: